OWNERS_ALIASES

```diff
@@ -44,3 +44,12 @@ aliases:
     - lance6716
     - tangenta
     - wjhuang2016
+  sig-approvers-table: # approvers for table packages.
+    - Benjamin2037
+    - D3Hunter
+    - gmhdbjd
+    - lance6716
```

Comment:
```suggestion
```

in fact I'm not familiar with this package

Reply:
OK.